### PR TITLE
Add common interface for name and parameters

### PR DESCRIPTION
**Pull request by IvanBorislavovDimitrov**
LMCROSSITXSADEPLOY-1685

### CODE REVIEW

File: com.sap.cloud.lm.sl.mta/src/main/java/com/sap/cloud/lm/sl/mta/model/NamedElementWithParametersContainer.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package com.sap.cloud.lm.sl.mta.model;
+
+public interface NamedElementWithParametersContainer extends NamedElement, ParametersContainer {
```
Comment (boyan-velinov) [inline com.sap.cloud.lm.sl.mta/src/main/java/com/sap/cloud/lm/sl/mta/model/NamedElementWithParametersContainer.java]: Probably other name, e.g. NamerParametersContainer 
How this sounds?

Comment (IvanBorislavovDimitrov) [reply com.sap.cloud.lm.sl.mta/src/main/java/com/sap/cloud/lm/sl/mta/model/NamedElementWithParametersContainer.java]: Renamed to -> NameParametersContainer

File: com.sap.cloud.lm.sl.mta/src/main/java/com/sap/cloud/lm/sl/mta/model/NameParametersContainer.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package com.sap.cloud.lm.sl.mta.model;
+
+public interface NameParametersContainer extends NamedElement, ParametersContainer {
```
Comment (boyan-velinov) [inline com.sap.cloud.lm.sl.mta/src/main/java/com/sap/cloud/lm/sl/mta/model/NameParametersContainer.java]: NamedParametersContainer

Comment (IvanBorislavovDimitrov) [reply com.sap.cloud.lm.sl.mta/src/main/java/com/sap/cloud/lm/sl/mta/model/NameParametersContainer.java]: Done

**Pull request merged**